nixos/tests/tracee.nix

```diff
@@ -7,11 +7,11 @@ import ./make-test-python.nix ({ pkgs, ... }: {
       # require docker/dockerd
       virtualisation.docker.enable = true;
 
-      environment.systemPackages = [
+      environment.systemPackages = with pkgs; [
```

Comment:
I'm not 100% sure but I think the goal is to reduce the use of `with`?

Reply:
*over big code parts and entire files

Reply:
In this case it's pretty clear which values are coming from the `with pkgs;` because there's no other `with`s and it's over only this array/list